content/01.abstract.md

```diff
@@ -2,11 +2,10 @@
 
 Understanding how dysregulated transcriptional processes result in tissue-specific pathology requires a mechanistic interpretation of expression regulation across different cell types.
 It has been shown that this insight is key for the development of new therapies.
-These mechanisms can be identified with transcriptome-wide association studies (TWAS), which have represented an important step forward to test the mediating role of gene expression in GWAS associations.
-However, due to pervasive eQTL sharing across tissues, TWAS has not been successful in identifying causal tissues, and other methods generally do not take advantage of the large amounts of RNA-seq data publicly available.
-Here we introduce PhenoPLIER, a polygenic approach that leverages modules of genes with shared expression patterns across large data compendia to project both gene-trait associations and pharmacological perturbation data into a common latent representation for a joint analysis.
-We observed that diseases were significantly associated with gene modules expressed in relevant cell types, such as hypothyroidism with T cells and thyroid.
-Our approach was more accurate in predicting known drug-disease pairs and identifying known mechanisms of action.
-Furthermore, using a CRISPR-screen for the analysis of lipid regulation, we show that PhenoPLIER is more robust to prioritize drug targets than a standard single-gene method.
-<!-- Our results also revealed stable trait clusters, including a complex branch involving lipids with cardiovascular, autoimmune, and neuropsychiatric disorders. -->
-By incorporating groups of co-expressed genes, PhenoPLIER can contextualize genetic associations and reveal potentially attractive targets for pharmacologic intervention when single disease-associated genes are not detected or are not druggable.
+These mechanisms can be identified with transcriptome-wide association studies (TWAS), which have represented a significant step forward to test the mediating role of gene expression in GWAS associations.
+However, it is hard to disentangle the causal tissues using eQTL data alone, and other methods generally do not take advantage of the large amounts of RNA-seq data publicly available.
+Here we introduce PhenoPLIER, a polygenic approach that maps both gene-trait associations and pharmacological perturbation data into a common latent representation for a joint analysis.
+This representation is based on modules of genes with similar expression patterns across the same tissues.
+We observed that diseases were significantly associated with gene modules expressed in relevant cell types, and our approach was accurate in predicting known drug-disease pairs and inferring mechanisms of action.
+Furthermore, using a CRISPR screen to analyze lipid regulation, we show that PhenoPLIER is more robust to prioritize drug targets than a standard single-gene method.
```

Comment:
```suggestion
Furthermore, using a CRISPR screen to analyze lipid regulation, we found that functionally important players lacked TWAS associations but prioritized in phenotype-associated modules by PhenoPLIER.
```

Reply:
Just in case, is it `but prioritized` or `but were prioritized`?

Reply:
yup 👍